interface/reports/external_data.php

```diff
@@ -149,8 +149,8 @@
             </div>
             <div class="row">
                 <div class="col-sm-12">
-                    <div class="dm-ed-in-3 dm-ed-in-5">
-                        <table width="100%;">
+                    <div class="dm-ed-in-3 dm-ed-in-5 table-responsive jumbotron py-4">
+                        <table class="table">
                             <tr class="dm-ed-in-9">
```

Comment:
Hey @tywrenn 

Looks like here use some of the custom layouts, Should I convert or remove it into the Bootstrap style?

Reply:
Yeah convert it over to the bootstrap style